content/geometry/object-human/human-structure.adoc

```diff
@@ -0,0 +1,861 @@
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing upwards
+
+| *z-axis*
+| Pointing sidewards
+|===
+
+NOTE: Please add a figure and description.
+
+== Neck 
+
+The `Neck` bone represents the most upper parts und bones of the spine, that is, the Cervical spine bones of the human skeleton.
+
+.Neck bone
+[%header, cols="20, 80"]
+|===
+
+2+^| Neck 
+
+| *Origin*
+| At the height of the lowest Cervical spine bone and in the middle of the geometry
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing upwards
+
+| *z-axis*
+| Pointing sidewards
+|===
+
+NOTE: Please add a figure and description.
+
+== Head
+
+The `Head` bone represents the head, that is, the skull of the human skeleton.
+
+.Head bone
+[%header, cols="20, 80"]
+|===
+
+2+^| Head
+
+| *Origin*
+| At the height of the first Cervical spine bone and in the middle of the geometry
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing upwards
+
+| *z-axis*
+| Pointing sidewards
+|===
+
+NOTE: Please add a figure and description.
+
+== Eye_Left
+
+The `Eye_Left` bone represents the left eye of the human body. It is used to calculate the eye level.
+
+.Eye_Left bone
+[%header, cols="20, 80"]
+|===
+
+2+^| Eye_Left
+
+| *Origin*
+| At the middle of the (eyeball) geometry
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing upwards
+
+| *z-axis*
+| Pointing sidewards
+|===
+
+NOTE: Please add a figure and description.
+
+== Eye_Right
+
+The `Eye_Right` bone represents the right eye of the human body. It is used to calculate the eye level.
+
+.Eye_Right bone
+[%header, cols="20, 80"]
+|===
+
+2+^| Eye_Right
+
+| *Origin*
+| At the middle of the (eyeball) geometry
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing upwards
+
+| *z-axis*
+| Pointing sidewards
+|===
+
+NOTE: Please add a figure and description.
+
+== Shoulder_Left 
+
+The `Shoulder_Left` bone represents the upper part of the left shoulder, that is, the interaction between the clavicle bone and the humerus head of the human skeleton.
+
+.Shoulder_Left bone
+[%header, cols="20, 80"]
+|===
+
+2+^| Shoulder_Left
+
+| *Origin*
+| At the height of the clavicle bone and in the middle of the geometry
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing upwards
+
+| *z-axis*
+| Pointing sidewards
+|===
+
+NOTE: Please add a figure and description.
+
+== Upper_Arm_Left
+
+The `Upper_Arm_Left` bone represents the upper part of the left arm, that is, the humerus head of the human skeleton.
+
+.Upper_Arm_Left bone
+[%header, cols="20, 80"]
+|===
+
+2+^| Upper_Arm_Left
+
+| *Origin*
+| At the height of the humerus head and in the middle of the geometry
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing upwards
+
+| *z-axis*
+| Pointing sidewards
+|===
+
+NOTE: Please add a figure and description.
+
+== Lower_Arm_Left
+
+The `Lower_Arm_Left` bone represents the lower part of the left arm, that is, the left elbow and Radius and Ulna of the human skeleton.
```

Comment:
Is it correct to add the bones here? Please also check for the legs accordingly.